Assets/MixedRealityToolkit/Utilities/MixedRealityPlayspace.cs

```diff
@@ -0,0 +1,130 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Utilities;
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    public static class MixedRealityPlayspace
```

Comment:
I like this new class a lot.

Reply:
You can thank @Alexees for most of it! I just did some minor reformatting and tweaking.

Reply:
Thanks @Alexees